docs/zh_CN/deployment/on-huawei-cloud.md

```diff
@@ -1,163 +1,164 @@
-csi-local-topology  everest-csi-provisioner         17d          # 本地持久卷
-```
+## 快速部署一个 EMQX 集群
 
-下面是 EMQX Custom Resource 的相关配置，你可以根据希望部署的 EMQX 的版本来选择对应的 APIVersion，具体的兼容性关系，请参考[EMQX Operator 兼容性](../README.md):
+下面是 EMQX 自定义资源的相关配置。你可以根据你想部署的 EMQX 版本选择相应的 APIVersion。关于具体的兼容性关系，请参考[ EMQX 与 EMQX Operator 的兼容性列表](../README.md)：
 
 :::: tabs type:card
-::: tab v2alpha1
-
-```yaml
-apiVersion: apps.emqx.io/v2alpha1
-kind: EMQX
-metadata:
-  name: emqx
-spec:
-  image: emqx:5.0
-  coreTemplate:
-    spec:
-      volumeClaimTemplates:
+::: tab apps.emqx.io/v1beta4
+
++ 将下面的内容保存成 YAML 文件，并通过 `kubectl apply` 命令部署它
+
+  ```yaml
+  apiVersion: apps.emqx.io/v1beta4
+  kind: EmqxEnterprise
+  metadata:
+    name: emqx-ee
+  spec:
+    ## EMQX 自定义资源不支持在运行时更新这个字段
+    persistent:
+      metadata:
+        name: emqx-ee
+      spec:
+        ## 更多内容：https://support.huaweicloud.com/usermanual-cce/cce_10_0380.html#section1
         storageClassName: csi-disk
         resources:
           requests:
             storage: 20Mi
         accessModes:
-        - ReadWriteOnce
-```
-:::
-::: tab v1beta4
-
-```yaml
-apiVersion: apps.emqx.io/v1beta4
-kind: EmqxEnterprise
-metadata:
-  name: emqx-ee
-spec:
-  persistent:
-    metadata:
-      name: emqx-ee
-    spec:
-      storageClassName: csi-disk
-      resources:
-        requests:
-          storage: 20Mi
-      accessModes:
-        - ReadWriteOnce
-  template:
-    spec:
-      emqxContainer:
-        image:
-          repository: emqx/emqx-ee
-          version: 4.4.14
-```
-:::
-::::
-
-## 通过 LoadBalancer 访问 EMQX 集群
-
-负载均衡( LoadBalancer )可以通过弹性负载均衡从公网访问到工作负载，与弹性 IP 方式相比提供了高可靠的保障，一般用于系统中需要暴露到公网的服务。负载均衡访问方式由公网弹性负载均衡服务地址以及设置的访问端口组成，例如“10.117.117.117:80”。关于更多负载均衡的内容，请查看[负载均衡(LoadBalancer)](https://support.huaweicloud.com/usermanual-cce/cce_10_0014.html)。
-
-在公有云中，一般通过配置资源的 Annotation 来配置负载均衡器的相关属性，
-
-修改 EMQX Custom Resource 的配置，添加相应的 Annotation，并将 Service Type 设置为 `LoadBalancer`，如下所示:
-
-:::: tabs type:card
-::: tab v2alpha1
-
-```yaml
-apiVersion: apps.emqx.io/v2alpha1
-kind: EMQX
-metadata:
-  name: emqx
-spec:
-  image: emqx:5.0
-  listenersServiceTemplate:
-    metadata:
-      annotations:
-        kubernetes.io/elb.class: union
-        kubernetes.io/elb.autocreate: |
-          {
-            "type": "public",
-            "name": "emqx",
-            "bandwidth_name": "cce-emqx",
-            "bandwidth_chargemode": "bandwidth",
-            "bandwidth_size": 5,
-            "bandwidth_sharetype": "PER",
-            "eip_type": "5_bgp"
-          }
-    spec:
-      type: LoadBalancer
-```
-:::
-::: tab v1beta4
+          - ReadWriteOnce
+    template:
+      spec:
+        emqxContainer:
+          image:
+            repository: emqx/emqx-ee
+            version: 4.4.14
+    serviceTemplate:
+      metadata:
+        annotations:
+          ## 自动创建关联的 ELB，详细字段说明请参考：https://support.huaweicloud.com/usermanual-cce/cce_10_0252.html#cce_10_0252__table268711532210
```

Comment:
```suggestion
          ## 自动创建关联的 ELB，详细字段说明请参考：https://support.huaweicloud.com/usermanual-cce/cce_10_0014.html#section4
```

Reply:
Check this document，looks the `kubernetes.io/elb.class` is request ?

Reply:
> Check this document，looks the `kubernetes.io/elb.class` is request ?

Not necessary, I verified